java/src/main/java/com/saucelabs/saucebindings/SauceMobileSession.java

```diff
@@ -0,0 +1,52 @@
+package com.saucelabs.saucebindings;
+
+import io.appium.java_client.AppiumDriver;
+import lombok.Getter;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.InvalidArgumentException;
+import org.openqa.selenium.MutableCapabilities;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class SauceMobileSession extends SauceSession {
```

Comment:
Not sure how I feel about Dividing `SauceSession` and `SauceMobileSession`. Can we combine the logic here into `SauceSession` and have a single `session` concept?

Reply:
We need some way of differentiating Appium Driver from Selenium Driver type. I started to look into doing generics and interfaces, but I can't tell if that's actually better for users.